### PR TITLE
Upgrade to 7.0.1

### PR DESCRIPTION
**Pull request by Caesarovich**
Upgrading engine to boxen [v7.0.1](https://github.com/sindresorhus/boxen/releases/tag/v7.0.1)

Also please make sure to remove the `upgrade` branch that is here for apparently no reason ;)

Have a nice day.

### CODE REVIEW

**Comment by Caesarovich** [comment]
@sindresorhus ;) (Also would you want to add me as maintainer there too ?)

File: readme.md [hunk 1]
```diff
@@ -15,37 +15,53 @@ npm install --global boxen-cli
...
+	  3~~~~~~~~~~4
+
+	  $ boxen --align=center --width=16 'Rainbows are so cool!'
+	  ┌──────────────┐
+	  │ Rainbows are │
+	  │   so cool!   │
+	  └──────────────┘
+
+	  $ boxen --title='Yes it is' --title-alignment=center 'Is that a centered title?'
+
```
Comment (sindresorhus) [inline readme.md]: There should not be an empty line here.

File: readme.md [hunk 2]
```diff
@@ -15,37 +15,53 @@ npm install --global boxen-cli
...
-    $ boxen --border-style=doubleSingle …like everyone
-    ╒══════════════╕
-    │…like everyone│
-    ╘══════════════╛
-
-    $ boxen --border-style='1234-~|║' ASCII ftw!
-    1----------2
-    |ASCII ftw!║
-    3~~~~~~~~~~4
+	Usage
```
Comment (sindresorhus) [inline readme.md]: Incorrect indentation on the new content here. It should be exactly like in the help output.

**Comment by sindresorhus** [comment]
I'm confused. Most of these changes were already done in https://github.com/sindresorhus/boxen-cli/commit/41a415a71a4c4de7493b32dda04df623c93d5fdb

**Comment by Caesarovich** [comment]
@sindresorhus Yup apparently I had messed up something in my branches

**Comment by sindresorhus** [comment]
This is not really needed though. Semver handles this for us. Read more here: https://nodesource.com/blog/semver-tilde-and-caret

**Comment by Caesarovich** [comment]
Fair well 👍

**Pull request closed without merging**